src/components/Step/Step.tsx

```diff
@@ -3,23 +3,33 @@ import React, { forwardRef, useEffect, useState } from 'react'
 import { CircularProgress } from '@/components/CircularProgress'
 import { SvgActionCheck, SvgActionLock, SvgActionTrash } from '@/components/_icons'
 
-import { Overhead, ProgressContainer, StepDetails, StepNumber, StepStatus, StepTitle, StepWrapper } from './Step.styles'
+import {
+  Overhead,
+  ProgressContainer,
+  StepDetails,
+  StepNumber,
+  StepStatus,
+  StepTitle,
+  StepType,
+  StepWrapper,
+} from './Step.styles'
 
+import { Text } from '../Text'
 import { IconButton } from '../_buttons/IconButton'
 
 export type StepProps = {
   title: string
   variant?: 'file' | 'default'
-  completed?: boolean
```

Comment:
I'm not sure we should combine `active` and `completed` props into `stepType`. I think they mean pretty different things, and even tho it might not be the case right now; it could very well be the case in the future

Reply:
hmm, I don't agree, for now, the design specifies that there will be only three possibilities and I think we should follow the design as much as possible.
![image](https://user-images.githubusercontent.com/51168865/150930637-558d0dae-25b2-45f1-aff3-9e1b6bbcc2b2.png)
If we actually need this in the future we can always add these props. We don't need to make this more complicated. 

I did a small change in naming, though. `type` is now either `file` or `default` and `variant` is now either `current`, `future`, or `completed`

Reply:
I think the naming is better for sure; whether we should follow designs as close as possible, I agree when it comes to styles, but logic wise, it is up to us to make sure it is a sound API; as we said, in this case, it doesn't matter too much but just something to keep in mind 👌